libs/providers/ofrep-web/src/lib/ofrep-web-provider.spec.ts

```diff
@@ -187,7 +190,9 @@ describe('OFREPWebProvider', () => {
       flagKey,
       value: false,
       errorCode: 'PARSE_ERROR',
-      errorMessage: 'parse error for flag key parse-error: custom error details',
+      // TODO: there's a bug in the web SDK which calls `instantiateErrorByErrorCode` without a message, so our message is lost :(
```

Comment:
:cry: 

https://github.com/open-feature/js-sdk/blob/f6bc6951a32fbbed9b35c1bdfd023b02874b87a1/packages/web/src/client/internal/open-feature-client.ts#L256

This should also pass the message, until it does we'll lose this error.